hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/commit/TestCopyOnWriteActionExecutor.java

```diff
@@ -532,7 +532,7 @@ public void testPartitionMetafileFormat(boolean partitionMetafileUseBaseFormat)
     writeClient.startCommitWithTime(instantTime);
 
     // Insert new records
-    final JavaRDD<HoodieRecord> inputRecords = generateTestRecordsForBulkInsert(jsc, 10);
+    final JavaRDD<HoodieRecord> inputRecords = generateTestRecordsForBulkInsert(jsc, 50);
     writeClient.bulkInsert(inputRecords, instantTime);
```

Comment:
ok would changing to 50 guarantee records generated for the required partitions? are we able to specify partitions to generate records?

Reply:
already we instantiate w/ 3 partitions. in all our tests we generate 50 to 100 records and it generally works out. just that, with 10 records, 1 in 25 to 30 runs, sometimes not all 3 partitions gets the record.

Reply:
after my fix, I ran it for 200 times locally and could not reproduce the flakiness.